addons/udes_stock/models/stock_picking_type.py

```diff
@@ -429,6 +429,12 @@ class StockPickingType(models.Model):
         default=False,
         help="If set, the user will be shown the list of all tasks during the picking workflow",
     )
+    u_batch_dest_loc_not_allowed = fields.Boolean(
+        string="Do not allow pickings with block destination location",
```

Comment:
This string is too long, won't display nicely on UI. Could you simplify the title? We always have the help string for a more in-depth explanation.

Also please remember to capitalise the string, capitalizemytitle.com is a useful tool for this. :)

Reply:
How about "No blocked location pickings" sounds to you?

Reply:
I think we need to include the fact it's for destination locations, how about `No Blocked Dest. Location Pickings`? Still a bit long but cut down a fair bit from before.

Might be worth getting another opinion if unsure.